### PR TITLE
Typings fix

### PR DESCRIPTION
**Pull request by PeterKottas**
Just fixed errors typescript was complaining about.

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export interface ServiceConfiguration {
 }
 
 export type BaseAuthConfiguration =
-  | {
```
Comment (kitten) [inline index.d.ts]: I'm pretty sure this was a style choice that @jevakallio introduced using an automated prettier format. Have you accidentally skipped our lint-staged formatting? https://github.com/FormidableLabs/react-native-app-auth/blob/master/package.json#L18-L19

File: index.d.ts [hunk 2]
```diff
@@ -17,13 +17,13 @@ export type BaseAuthConfiguration =
       serviceConfiguration?: ServiceConfiguration;
     };
 
-export interface AuthConfiguration extends BaseAuthConfiguration {
+export type AuthConfiguration = BaseAuthConfiguration & {
```
Comment (kitten) [inline index.d.ts]: This looks good to me. Good catch!

File: index.d.ts (hunk 1 above)
Comment (PeterKottas) [reply index.d.ts]: Ah ok, I see. Never seen it written like that, it reads like "nothing or this or that" to me so I was afraid if it wouldn't accept nothing the way it was. Hope that makes sense :D Anyways, I've reverted it. Rest should be good to go in.

**Comment by PeterKottas** [comment]
👍

**Pull request merged**